services/cd-service/pkg/repository/repository.go

```diff
@@ -126,6 +129,21 @@ func openOrCreate(path string) (*git.Repository, error) {
 			return nil, err
 		}
 	}
+	if !disableSqlite {
+		sqlitePath := filepath.Join(path, "odb.sqlite")
+		be, err := sqlitestore.NewOdbBackend(sqlitePath)
+		if err != nil {
+			return nil, fmt.Errorf("creating odb backend: %w", err)
+		}
+		odb, err := repo2.Odb()
+		if err != nil {
+			return nil, fmt.Errorf("gettting odb: %w", err)
+		}
+		err = odb.AddBackend(be, 99)
```

Comment:
don't we need to remove a backend too in this case?

Reply:
No we can't. Removing a backend is not supported atm

Reply:
Added a comment here.